scripts/linters/pylint_extensions_test.py

```diff
@@ -2909,3 +2742,114 @@ def func():
 
         with self.checker_test_object.assertAddsMessages(message):
             temp_file.close()
+
+
+class SingleSpaceAfterKeyWordCheckerTests(unittest.TestCase):
+
+    def setUp(self):
+        super(SingleSpaceAfterKeyWordCheckerTests, self).setUp()
+        self.checker_test_object = testutils.CheckerTestCase()
+        self.checker_test_object.CHECKER_CLASS = (
+            pylint_extensions.SingleSpaceAfterKeyWordChecker)
+        self.checker_test_object.setup_method()
+
+    def test_no_space_after_keyword(self):
+        node_no_space_after_keyword = astroid.scoped_nodes.Module(
+            name='test',
+            doc='Custom test')
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+
+        with python_utils.open_file(filename, 'w') as tmp:
+            tmp.write(
+                u"""
+                if(False):
+                    pass
+                elif(True):
+                    pass
+                while(True):
```

Comment:
Should we have separate tests for the while and yield keywords? (I mean separate test function.)

Reply:
Do we think that would be too repetitive? Compressing everything down to a single test function seems handy, but open to suggestions if we think separate test functions would be better!

Reply:
I'm fine with this considering our new check is more generic and should handle any keyword, so having one test is fine!